skbeam/core/tests/test_recip.py

```diff
@@ -136,6 +138,28 @@ def test_hkl_to_q():
     npt.assert_array_almost_equal(b_norm, recip.hkl_to_q(b))
 
 
+def test_gisaxs():
+    incident_beam = (1.0, 1.0)
+    reflected_beam = (3.0, 3.0)
+    pixel_size = (1.0, 1.0)
+    detector_size = (5, 5)
```

Comment:
Can you test with a rectangular detector instead of a square one?  Testing with a rectangle instead of a square is a great way to catch transpose errors (which I make All. The. Time.)